tests/api/test_order.py

```diff
@@ -174,6 +174,23 @@ def test_order_query(
     assert expected_method.type.upper() == method['type']
 
 
+def test_order_query_customer(api_client):
```

Comment:
Will ```test_order_query_by_customer``` be more accurate?

Reply:
That's not a big deal, the current name is also fine.